modules/df/df_tools/df_tools_media/modules/df_tools_gallery/config/install/core.entity_form_display.entity_gallery.media.default.yml

```diff
@@ -3,15 +3,15 @@ langcode: en
 status: true
 dependencies:
   config:
-    - entity_browser.browser.media_browser_in_modal
     - entity_gallery.type.media
     - field.field.entity_gallery.media.entity_gallery_media
     - field.field.entity_gallery.media.field_tags
   module:
-    - entity_browser
+    - content_moderation
+    - datetime
     - path
 _core:
-  default_config_hash: i6NixYXdxOaeNhbC_WM4OEC9MburFqVb5jMN-tSoB_w
+  default_config_hash: N5SgKxUwNaAsQGi--cawfjVnUwJMQnW9zFKbYFEJTOg
```

Comment:
This line should actually be removed, whoever created the original file must have forgotten to delete it.